weave/scorers/hallucination_scorer.py

```diff
@@ -151,8 +151,11 @@ def score(self, output: str, context: str) -> HallucinationResponse:
                 },
             ],
             model=self.model_id,
-            response_model=HallucinationResponse,
+            response_format=HallucinationResponse,
             temperature=self.temperature,
             max_tokens=self.max_tokens,
         )
-        return response.model_dump()  # Morgan wants this to be a dict
+        response = HallucinationResponse.model_validate_json(
+            response.choices[0].message.content
+        )
```

Comment:
Why do you need to do this?

Reply:
because it doesn't return a Pydantic object